assets/js/pages/criarpost.js

```diff
@@ -0,0 +1,20 @@
+function PostarFoto() {
+    let f = document.getElementById("fotoPost")
+    let l = document.getElementById("legendaPost")
+    let legenda = l.value
+
+    if(f.value.trim() == '' || l.value.trim() == ''){
+        document.getElementById('AlertaPreencherPost').classList.remove("d-none")
+        return
+    }
+
+    let UsuarioLogado = JSON.parse(sessionStorage.getItem("UsuarioLogado"))
+    let novoPost = {usuario: UsuarioLogado.Usuario, legenda: legenda}
+
+    sessionStorage.setItem("novoPost", JSON.stringify(novoPost))
+    sessionStorage.setItem("PostSucesso", "true")
```

Comment:
o que acha de criar uma constante no global.js para deixar todos os nomes usados no sessionStorage? assim:
- fica o nome centralizado
- nao corre risco de em um .js escrever de um jeito e no outro escrever de um jeito diferente

segue um exemplo:

no global.js
```
  const STORAGE_CHAVES = {
    usuarioLogado: 'UsuarioLogado',
    novoPost: 'novoPost',
    postSucesso: 'PostSucesso',
  };
```

nos outros .js
```suggestion
    sessionStorage.setItem(STORAGE_KEYS.novoPost, JSON.stringify(novoPost))
    sessionStorage.setItem(STORAGE_KEYS.postSucesso, "true")
```

o que acha?